lib/components/episode/episode_info.dart

```diff
@@ -4,26 +4,35 @@ import 'package:brunstadtv_app/helpers/ui/svg_icons.dart';
 import 'package:brunstadtv_app/helpers/utils.dart';
 import 'package:brunstadtv_app/helpers/widget_keys.dart';
 import 'package:brunstadtv_app/l10n/app_localizations.dart';
-import 'package:brunstadtv_app/theme/design_system/design_system.dart';
-
 import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
 import 'package:flutter_svg/svg.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
 
+import '../../api/brunstadtv.dart';
+import '../../theme/design_system/design_system.dart';
 import '../text_collapsible.dart';
 
-class EpisodeInfo extends StatelessWidget {
+class EpisodeInfo extends HookConsumerWidget {
   const EpisodeInfo(this.episode, {super.key, required this.onShareVideoTapped, this.extraChildren});
 
   final Query$FetchEpisode$episode episode;
   final void Function() onShareVideoTapped;
   final List<Widget>? extraChildren;
 
   @override
-  Widget build(BuildContext context) {
+  Widget build(BuildContext context, WidgetRef ref) {
     const showEpisodeNumber = false;
     final episodeNumberFormatted = '${S.of(context).seasonLetter}${episode.season?.number}:${S.of(context).episodeLetter}${episode.number}';
-
     final design = DesignSystem.of(context);
+
+    final inMyList = useState(episode.inMyList);
+
+    useEffect(() {
+      inMyList.value = episode.inMyList;
+      return null;
+    }, [episode.id]);
```

Comment:
```suggestion
    }, [episode.id, episode.inMyList]);
```
because inMyList can change for the same id if we were to refetch the episode